discretisedfield/mesh.py

```diff
@@ -950,6 +950,179 @@ def plane(self, *args, n=None, **kwargs):
 
         return plane_mesh
 
+    def sel(self, *args, **kwargs):
+        """Select a part of the mesh.
+
+        If one of the axis from ``region.dims`` is passed as a string, a mesh of a
+        reduced dimension along the axis and perpendicular to it is extracted,
```

Comment:
I find this very difficult to understand. I will think about alternatives.

Reply:
Yeah... I found it a bit dificult to explain all the features in the docstrings. It would be good in you have better suggestions.